aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageUploadAccessLevelTest.java

```diff
@@ -80,6 +81,17 @@ public static void setUpOnce() throws AmplifyException, InterruptedException {
         storage = SynchronousStorage.delegatingTo(asyncDelegate);
     }
 
+    /**
+     * Reset all the assigned static fields.
+     */
+    @AfterClass
```

Comment:
why is this added?

Reply:
I asked Sunil about this -- it's not explicitly related but nice to have